includes/Checker/Checks/Abstract_PHP_CodeSniffer_Check.php

```diff
@@ -92,6 +92,14 @@ final public function run( Check_Result $result ) {
 			$defaults[] = '--ignore=*/' . implode( '/*,*/', $directories_to_ignore ) . '/*';
 		}
 
+		// Set the Minimum WP version supported for the plugin.
+		if ( $result->plugin()->minimum_supported_wp() ) {
+			// Due to the syntax of runtime-set, these must be passed as individual args.
+			$defaults[] = '--runtime-set';
+			$defaults[] = 'minimum_wp_version';
+			$defaults[] = $result->plugin()->minimum_supported_wp();
```

Comment:
This should probably be wrapped..
```suggestion
			$defaults[] = escapeshellarg( $result->plugin()->minimum_supported_wp() );
```

For context of the syntax requiring individual args, this is handled here:
https://github.com/squizlabs/PHP_CodeSniffer/blob/7763e2e1f773cb0615ed8afa133189fc804f583d/src/Config.php#L828-L847 which doesn't have a handler for `--runtime-set=...` in the `default:`: https://github.com/squizlabs/PHP_CodeSniffer/blob/7763e2e1f773cb0615ed8afa133189fc804f583d/src/Config.php#L848-L849

Reply:
Not sure whether it's needed 🤔 

PHPCS isn't run in command line context in this branch, so I don't think there's any risk for executing other command line commands here. I believe the only risk from malicious data here would be to cause a failure in PHPCS.

I think it's okay to leave it out for now, as I'm unsure whether there are any side effects from adding it.